src/components/add-note-form/AddNoteForm.jsx

```diff
@@ -84,18 +90,21 @@ export const AddNoteForm = ({ name }) => {
           <input
               onChange={onChangeHandler}
               type="text"
-              className="form-control"
+              className={inputClasses}
               name="description"
               value={formState.description}
               placeholder="Описание"
           />
         </div>
-        <div className="">
           <button type="submit" className={btnSubmitClasses}>Добавить</button>
-        </div>
-        <div className="">
-          <button onClick={() => setIsAddingNote(false)} type="button" className={cancelBtnClasses}>Отмена</button>
-        </div>
+          <button
+              onTouchStart={() => setIsAddingNote(true)}
+              onClick={() => setIsAddingNote(false)}
```

Comment:
why are you using different values for the same functionality for onTouchStart/onClick => true/false ?